DOCS-#2635: Refactor troubleshooting docs and add FAQs (#3816)
modin-project/modin: docs/getting_started/troubleshooting.rst

===============

We hope your experience with Modin is bug-free, but there are some quirks about Modin

> that may require troubleshooting. If you still are having issues, contact us via

Reviewer: ```suggestion
that may require troubleshooting. If you are still having issues, please post on 
```

> one of the methods listed on this page_ or file a Github issue_.

Reviewer: ```suggestion
the #support channel on our Slack_ community or open a Github issue_.
```

  pd_df.index.name = None

> 

Reviewer: ```suggestion

.. _Slack: https://modin.org/slack.html
```
